examples/label_studio_annotation/steps/load_image_data_step.py

```diff
@@ -0,0 +1,45 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+import glob
+import os
+from typing import Dict
+
+from PIL import Image
+
+from zenml.steps import Output, step
+from zenml.steps.base_step_config import BaseStepConfig
+from zenml.steps.step_context import StepContext
+
+
+class LoadImageDataConfig(BaseStepConfig):
+    base_path = os.path.join(os.getcwd(), "data")
```

Comment:
I'd not make it relative to the current working directory, as that might be dynamic. Instead relative to this file is safer IMO

Reply:
This seems to be a philosophical difference between you and @fa9r. I'll make it a relative link.